zubhub_frontend/zubhub/src/views/create_activity/CreateEditActivity.jsx

```diff
@@ -203,7 +214,9 @@ export default function CreateActivity(props) {
       <Box className={classes.formContainer}>
         <Grid item md={12} lg={12}>
           <Box sx={{ textAlign: isSmallScreen ? 'left' : 'center' }}>
-            <Typography className={clsx(commonClasses.title1)}>Create Activity</Typography>
+            <Typography className={clsx(commonClasses.title1)}>
+              {editting ? 'Edit Activity' : 'Create Activity'}
```

Comment:
can we have translation implemented here?

Reply:
On it. Thanks